### PR TITLE
Add ui templates

### PR DESCRIPTION
**Pull request by tomgeorge**
#### What is this PR About?
Describe the contents of the PR

#### How do we test this?
Provide commands/steps to test this PR.

cc: @redhat-cop/containerize-it

### CODE REVIEW

File: hygieia/.applier/group_vars/seed-hosts.yml [hunk 1]
```diff
@@ -33,6 +34,18 @@ openshift_cluster_content:
       tags:
         - unpriv
         - all
+    - name: UI Builder Image
```
Comment (etsauer) [inline hygieia/.applier/group_vars/seed-hosts.yml]: @tomgeorge could you combine these two builds into a single template, consistent with the others?

File: hygieia/.applier/group_vars/seed-hosts.yml [hunk 2]
```diff
@@ -54,3 +67,28 @@ openshift_cluster_content:
       tags:
         - unpriv
         - all
+    - name: UI
```
Comment (etsauer) [inline hygieia/.applier/group_vars/seed-hosts.yml:64]: same here. I think just one template called `deployment-ui`

**Review by etsauer — CHANGES_REQUESTED** [review]
@tomgeorge

**Comment by tomgeorge** [comment]
@etsauer Consolidated the images and renamed `run-ui` to just `ui`.  Note this PR does not have API connectivity, that one is in #4

**Review by etsauer — APPROVED** [review]

**Pull request merged**